jupyter_remote_desktop_proxy/install.py

```diff
@@ -0,0 +1,151 @@
+import argparse
+import hashlib
+import os
+import platform
+import shutil
+import tarfile
+import textwrap
+from urllib.request import HTTPError, urlopen, urlretrieve
+
+from xml.etree import ElementTree
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+def checksum_file(path):
+    """Compute the md5 checksum of a path"""
+    hasher = hashlib.md5()
+    with open(path, "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hasher.update(chunk)
+    return hasher.hexdigest()
+
+def fetch_release_info(tigervnc_archive, tigervnc_version):
+    """Fetch the download url and md5 checksum from tigervnc release rss"""
+    url = (
+        "https://sourceforge.net/projects/tigervnc/rss"
+        f"?path=/stable/{tigervnc_version}"
+    )
+    release_title = f'/stable/{tigervnc_version}/{tigervnc_archive}'
+    print(f"Fetching md5 and download url from {url}")
+
+    with urlopen(url) as f:
+        tree = ElementTree.parse(f)
+        root = tree.getroot()
+        channel = root.find('channel')
+        for item in channel.iterfind('item'):
+            title = item.find('title')
+            if title.text == release_title:
+                media = item.find('{http://video.search.yahoo.com/mrss/}content')
+                url = media.get('url')
+                md5 = media.find('{http://video.search.yahoo.com/mrss/}hash').text
+                break
+        else:
+            raise Exception(f"Could not find a tarball \"{tigervnc_archive}\" on sourceforge.net")
+    return url, md5
+
+
+def install_tigervnc(prefix, plat, tigervnc_version):
+    tigervnc_path = os.path.join(prefix, "tigervnc")
+
+    tigervnc_archive = f"tigervnc-{tigervnc_version}.{plat}.tar.gz"
+    tigervnc_archive_path = os.path.join(prefix, tigervnc_archive)
+
+    if os.path.exists(tigervnc_path):
```

Comment:
This check doesn't seem to be working. If I try to install tigervnc twice:
```
# python -mjupyter_remote_desktop_proxy.install 

Using existing output directory /usr/local/lib/python3.11/site-packages/jupyter_remote_desktop_proxy/share/tigervnc...
Fetching md5 and download url from https://sourceforge.net/projects/tigervnc/rss?path=/stable/1.10.0
Downloading tigervnc 1.10.0 from https://sourceforge.net/projects/tigervnc/files/stable/1.10.0/tigervnc-1.10.0.x86_64.tar.gz/download...
Extracting the archive...
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.11/site-packages/jupyter_remote_desktop_proxy/install.py", line 151, in <module>
    main()
  File "/usr/local/lib/python3.11/site-packages/jupyter_remote_desktop_proxy/install.py", line 147, in main
    install_tigervnc(deps_dir, plat, tigervnc_version)
  File "/usr/local/lib/python3.11/site-packages/jupyter_remote_desktop_proxy/install.py", line 78, in install_tigervnc
    shutil.move(f"{prefix}/tigervnc-{tigervnc_version}.{plat}/usr/bin", prefix)
  File "/usr/local/lib/python3.11/shutil.py", line 823, in move
    raise Error("Destination path '%s' already exists" % real_dst)
shutil.Error: Destination path '/usr/local/lib/python3.11/site-packages/jupyter_remote_desktop_proxy/share/tigervnc/bin' already exists
```

Reply:
The variable was incorrectly configured. I have fixed it.